jenkins/ci.suse.de/pipelines/openstack-ardana-gerrit.Jenkinsfile

```diff
@@ -87,6 +87,8 @@ The following links can also be used to track the results:
               string(name: 'controllers', value: "2"),
               string(name: 'sles_computes', value: "1"),
               string(name: 'cloudsource', value: "$cloudsource"),
+              string(name: 'ses_enabled', value: "$ses_enabled"),
```

Comment:
The `openstack-ardana-gerrit` JJB template doesn't/shouldn't have `ses_enabled` and `ses_rgw_enabled` parameters. This should just be enabled here, i.e.:

```
string(name: 'ses_enabled', value: "true"),
string(name: 'ses_rgw_enabled', value: "false"),
```

Reply:
Wouldn't this also enable SES for cloud9?

Reply:
Yeah , you're right, I missed that.